cmd/list/region/cmd.go

```diff
@@ -110,15 +134,29 @@ func run(cmd *cobra.Command, _ []string) {
 
 	// Create the writer that will be used to print the tabulated results:
 	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
-	fmt.Fprintf(writer, "ID\t\tNAME\t\tMULTI-AZ SUPPORT\n")
+	headerFormat := "ID\t\tNAME\t\tMULTI-AZ SUPPORT\n"
+	if hypershiftEnabled {
+		headerFormat = "ID\t\tNAME\t\tMULTI-AZ SUPPORT\t\tHOSTED-CP SUPPORT\n"
```

Comment:
@wgordon17 do we need to display Hosted-CP to the user or Hypershift Enabled??

Reply:
`HOSTED-CP` since `HyperShift` isn't meant for customer facing usage